www/class/centreonWidget/Params/Connector/Poller.class.php

```diff
@@ -89,6 +89,7 @@ public function getListValues($paramId)
                     );
                 }
 
+                $tab = [];
```

Comment:
doesn't work on 21.10 at least 

```suggestion
                $tab = [null => null];
```